xml/System.Formats.Tar/TarEntry.xml

```diff
@@ -68,22 +60,10 @@ All the properties exposed by this class are supported by the <xref:System.Forma
           <para>Gets a stream that represents the data section of this entry.</para>
           <para>Sets a new stream that represents the data section, if it makes sense for the <see cref="P:System.Formats.Tar.TarEntry.EntryType" /> to contain data. If a stream already exists, the old stream is disposed before substituting it with the new stream. Setting a <see langword="null" /> stream is allowed.</para>
         </value>
-        <remarks>
-          <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-If you write data to this data stream, make sure to rewind it to the desired start position before writing this entry into an archive using <xref:System.Formats.Tar.TarWriter.WriteEntry(System.Formats.Tar.TarEntry)>.
-
-          ]]></format>
-        </remarks>
+        <remarks>If you write data to this data stream, make sure to rewind it to the desired start position before writing this entry into an archive using <see cref="M:System.Formats.Tar.TarWriter.WriteEntry(System.Formats.Tar.TarEntry)" />.</remarks>
         <exception cref="T:System.InvalidOperationException">Setting a data section is not supported because the <see cref="P:System.Formats.Tar.TarEntry.EntryType" /> is not <see cref="F:System.Formats.Tar.TarEntryType.RegularFile" /> (or <see cref="F:System.Formats.Tar.TarEntryType.V7RegularFile" /> for an archive of <see cref="F:System.Formats.Tar.TarEntryFormat.V7" /> format).</exception>
-        <exception cref="T:System.IO.IOException">
-Cannot set an unreadable stream.
-
--or-
-
-An I/O problem occurred.</exception>
+        <exception cref="T:System.ArgumentException">Cannot set an unreadable stream.</exception>
```

Comment:
```suggestion
        <exception cref="T:System.ArgumentException">The stream is unreadable.</exception>
```

Reply:
Oh, I got different suggestions the other day, based on the fact that all exception messages should begin as if they started (but not including): "The exception that is thrown when...". We came up with:

```suggestion
        <exception cref="T:System.ArgumentException">The stream does not support reading.</exception>

```

Reply:
Hope you don't mind if I use my suggestion instead. It would match all the other similar exceptions.